app/config.py

```diff
@@ -1,8 +1,19 @@
 import os
 import getpass
+import yaml
+
+def getcfg(path):
+    with open(path, "r") as stream:
+        try:
+            return yaml.safe_load(stream)
+        except yaml.YAMLError as exc:
+            print(exc)
+
 
 class DefaultConfig(object):
     AGORA_PATH = os.getenv('AGORA_PATH', os.path.join('/home', getpass.getuser(), 'agora'))
+    # For evan's sake
+    YAML_CONFIG = getcfg(os.path.join(AGORA_PATH, 'gardens.yaml'))
```

Comment:
Would you mind making this REPOS_CONFIG? Optionally keeping also 'YAML' in if you want that information.